target/linux/ipq806x/image/Makefile

```diff
@@ -72,6 +72,20 @@ define Device/ZyXELImage
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to $$$${BLOCKSIZE} | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
+define Device/buffalo_wxr-2533dhp
+	$(call Device/LegacyImage)
+	DEVICE_DTS := qcom-ipq8064-wxr-2533dhp
+	BOARD_NAME := wxr-2533dhp
+	DEVICE_TITLE := Buffalo WXR-2533DHP
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	IMAGE_SIZE := 65536k
+	KERNEL_IN_UBI := 1
+	IMAGES := sysupgrade.bin
+	DEVICE_PACKAGES := ath10k-firmware-qca99x0-ct squashfs-tools-mksquashfs
```

Comment:
In your flashing instructions you state that the user has to flash via the initramfs and then use sysupgrade.
Problem is the initramfs image is not enabled by default for the ipq806x targets since it does not have the ramdisk feature in the target's makefile. 

If this got fixed and FEATURES is extended . Then unfortunately, the initramfs images that the OpenWrt buildbots are creating for the official images (CONFIG_TARGET_ALL_PROFILES!) will not have the mksquashfs tool since the initramfs are being build to include the target's DEFAULT_PACKAGES and not the individual DEVICE_PACKAGES.

Reply:
I see. Can I add ```ramdisk``` to ```FEATURES``` in the Makefile?

Reply:
I see, OK.

Reply:
I sent the patch to the mailing list.